.cirrus.yml

```diff
@@ -212,14 +213,14 @@ task:
     memory: 24G
   env:
     << : *CIRRUS_EPHEMERAL_WORKER_TEMPLATE_ENV
-    MAKEJOBS: "-j8"
     FILE_ENV: "./ci/test/00_setup_env_native_tsan.sh"
 
 task:
   name: '[MSan, depends] [focal]'
   << : *GLOBAL_TASK_TEMPLATE
   container:
     image: ubuntu:focal
+    greedy: false
```

Comment:
I think this can be removed. I was wrong yesterday to say that `greedy` can lead to OOM. It can not, since we always set the jobs with `MAKEJOBS`. So only `MAKEJOBS` can lead to OOM.

Reply:
Done